### PR TITLE
Heatmap: update heatmap documentation

### PR DESCRIPTION
**Pull request by ryantxu**
In 9.1, we have swapped to a new heatmap panel -- this updates the docs for that panel

### CODE REVIEW

File: docs/sources/visualizations/heatmap.md [hunk 1]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
 
 ![](/static/img/docs/v43/heatmap_panel_cover.jpg)
 
-## Axes options
+## Calculate from data
 
-Use these settings to adjust how axes are displayed in your visualization.
+This will toggle if the data is already a calculated heatmap (from the data source/transformer), or one that should be
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:23]:
```suggestion
This setting determines if the data is already a calculated heatmap (from the data source/transformer), or one that should be
```

File: docs/sources/visualizations/heatmap.md [hunk 2]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
-- **Scale -** The scale to use for the Y axis value.
-  - **linear -** Linear scale.
-  - **log (base 2) -** Logarithmic scale with base 2.
-  - **log (base 10) -** Logarithmic scale with base 10.
-  - **log (base 32) -** Logarithmic scale with base 32.
-  - **log (base 1024) -** Logarithmic scale with base 1024.
-- **Y-Min -** The minimum Y value (default auto).
-- **Y-Max -** The maximum Y value (default auto).
-- **Decimals -** Number of decimals to render Y axis values with (default auto).
+Specify how the axis will be split into buckets. You can specify a time interval in the Size input, for example, the time range 1h. This will make the cells 1h wide on the X-axis.
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:28]:
```suggestion
This setting determines how the axis is split into buckets. You can specify a time interval in the **Size** input. For example, a time range of `1h` makes the cells 1-hour wide on the X-axis.
```

File: docs/sources/visualizations/heatmap.md [hunk 3]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
 
-### Buckets
+### Y Bucket
 
-> **Note:** If the data format is **Time series buckets**, then this section will not be available.
+Specify how the axis will be split into buckets.
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:32]:
```suggestion
This setting determines how the Y-axis is split into buckets.
```

File: docs/sources/visualizations/heatmap.md [hunk 4]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
 
-- **Y Axis Buckets -** Number of buckets Y axis will be split into.
-- **Size -** (Only visible if **Scale** is _linear_). Size of each Y axis bucket. This option has priority over **Y Axis Buckets**.
-- **Split Factor -** (Only visible if **Scale** is _log (base 2)_ or greater). By default Grafana splits Y values by log base. This option allows to split each default bucket into specified number of buckets.
-- **X Axis Buckets -** Number of buckets X axis will be split into.
-- **Size -** Size of each X axis bucket. Number or time interval (10s, 5m, 1h, etc). Supported intervals: ms, s, m, h, d, w, M, y. This option has priority over **X Axis Buckets**.
+### Y Bucket scale
 
-#### Bucket bound
+The scale to use for the Y axis value.
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:36]:
```suggestion
Select one of the following Y-axis value scales:
```

File: docs/sources/visualizations/heatmap.md [hunk 5]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
-- **Time series buckets -** Each time series already represents a Y-Axis bucket. The time series name (alias) needs to be a numeric value representing the upper or lower interval for the bucket. Grafana does no bucketing, so the bucket size options are hidden.
+### Unit
 
-## Display options
+Unit configuration
 
-Use these settings to refine your visualization.
+### Decimals
+
+Decimals configuration
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:58]:
```suggestion
This setting determines decimal configuration.
```

File: docs/sources/visualizations/heatmap.md [hunk 6]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
+
+### Min/Max value
+
+Configures the axis range
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:62]:
```suggestion
This setting configures the axis range.
```

File: docs/sources/visualizations/heatmap.md [hunk 7]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
 
-### Colors
+### Reverse
+
+When selected, the axis will be displayed in reverse order
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:66]:
```suggestion
When selected, the axis appears in reverse order.
```

File: docs/sources/visualizations/heatmap.md [hunk 8]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
       - **linear -** Linear scale. Bucket value maps linearly to the opacity.
       - **sqrt -** Power scale. Cell opacity calculated as `value ^ k`, where `k` is a configured **Exponent** value. If exponent is less than `1`, you will get a logarithmic scale. If exponent is greater than `1`, you will get an exponential scale. In case of `1`, scale will be the same as linear.
     - **Exponent -** value of the exponent, greater than `0`.
-  - **spectrum -** Bucket value represented by cell color.
-    - **Scheme -** If the mode is **spectrum**, then select a color scheme.
 
-### Color scale
+### Start/end color from value
 
 By default, Grafana calculates cell colors based on minimum and maximum buckets values. With Min and Max you can overwrite those values. Think of a bucket value as a Z-axis and Min and Max as Z-Min and Z-Max respectively.
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:86]:
```suggestion
By default, Grafana calculates cell colors based on minimum and maximum bucket values. With Min and Max you can overwrite those values. Consider a bucket value as a Z-axis and Min and Max as Z-Min and Z-Max, respectively.
```

File: docs/sources/visualizations/heatmap.md [hunk 9]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
 
-- **Min -** Minimum value using for cell color calculation. If the bucket value is less than Min, then it is mapped to the "minimum" color. Default is series min value.
-- **Max -** Maximum value using for cell color calculation. If the bucket value is greater than Max, then it is mapped to the "maximum" color. Default is series max value.
-
-### Legend
+- **Start -** Minimum value using for cell color calculation. If the bucket value is less than Min, then it is mapped to the "minimum" color. Default is series min value.
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:88]:
```suggestion
- **Start -** Minimum value using for cell color calculation. If the bucket value is less than Min, then it is mapped to the "minimum" color. The series min value is the default value.
```

File: docs/sources/visualizations/heatmap.md [hunk 10]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
+- **End -** Maximum value using for cell color calculation. If the bucket value is greater than Max, then it is mapped to the "maximum" color. Default is series max value.
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:89]:
```suggestion
- **End -** Maximum value using for cell color calculation. If the bucket value is greater than Max, then it is mapped to the "maximum" color. The series max value is the default value.
```

File: docs/sources/visualizations/heatmap.md [hunk 11]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
-- **Space -** Space in pixels between cells. Default is 1 pixel.
-- **Round -** Cell roundness in pixels. Default is 0.
+## Additional display options
 
 ### Tooltip
 
 - **Show tooltip -** Show heatmap tooltip.
-- **Histogram -** Show Y axis histogram on the tooltip. Histogram represents distribution of the bucket values for the specific timestamp.
-- **Decimals -** Number of decimals to render bucket value with (default auto).
+- **Show Histogram -** Show Y axis histogram on the tooltip. Histogram represents distribution of the bucket values for the specific timestamp.
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:100]:
```suggestion
- **Show Histogram -** Show a Y-axis histogram on the tooltip. A histogram represents the distribution of the bucket values for a specific timestamp.
```

File: docs/sources/visualizations/heatmap.md [hunk 12]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
+
+### Legend
+
+Choose whether to display the heatmap legend on the visualization or not.
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:104]:
```suggestion
Choose whether you want to display the heatmap legend on the visualization.
```

File: docs/sources/visualizations/heatmap.md [hunk 13]
```diff
@@ -18,87 +18,91 @@ The Heatmap panel visualization allows you to view histograms over time. For mor
...
+
+### Exemplars
+
+Set the color used to show exemplar data
```
Comment (chri2547) [inline docs/sources/visualizations/heatmap.md:108]:
```suggestion
Set the color used to show exemplar data.
```

**Review by chri2547 — CHANGES_REQUESTED** [review]
Added suggestions.

**Review by chri2547 — COMMENTED** [review]
The docs have been reviewed and are approved. Please go ahead and merge after Engr review and approval.

**Comment by ryantxu** [comment]
Thanks @chri2547 -- I applied all your changes, and you double check and approve?

(Automerge is enabled)

**Review by chri2547 — APPROVED** [review]

**Pull request merged**